### DISCUSSION TITLE
Inclusion in Mint base releases?

### DISCUSSION THREAD

**Discussion by AtomicRobotMan0101** [cboxdoerfer/fsearch#473] [Q&A]
Can I suggest that the dev makes a push to include fsearch in Mints distros/releases?

Its such an excellent tool for searching that it behooves us not to try.

I'm a reformed Windows user. One of the major issues I faced, in the way I worked, was finding something amongst my blizzard of files. "Everything" solved that. I moved to Linux Mint Cinnamon and lloovvveedd it.... BUT one of the things I missed the most was "Everything".

Fsearch solidly fills that gap. A glorious day when I found it!

The base Mint distro doesnt include a funky search. One is reliant on `find/locate/dbupdate`, which to a n00b, are hideous. 

Fsearch does everything a GUI user will want.

Hence my "request" to attempt to include it in the distro. This is not something I can do as a user, bug reporter and commenter on their boards, for I dont have the skills needed to accomodate their needs, but feel strongly that it should be included ( https://linuxmint-developer-guide.readthedocs.io/en/latest/ ).

**Comment by cboxdoerfer** [comment]
The most efficient and official way to get FSearch into Linux Mint is through Debian, because Linux Mint is using the Ubuntu repositories and Ubuntu in turn is based off Debian. So once FSearch gets included into Debian it eventually will show up in the Linux Mint repositories as well.

There has already been a request for that, but unfortunately nothing came of it yet: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=908891

I guess we simply need to wait until someone shows up and wants to package and maintain FSearch for Debian.

**Comment by AtomicRobotMan0101** [comment]
I believe it can be made into one of their own stand-alone packages as a "Mint Tools" or an "XApp" , for example these:  

https://linuxmint-developer-guide.readthedocs.io/en/latest/mint-tools.html#
https://linuxmint-developer-guide.readthedocs.io/en/latest/xapps.html

It may be a stupid idea, but I felt it important.
